tools/pcmsolver.py.in

```diff
@@ -373,7 +373,7 @@ def setup_keywords():
     green.add_kw('SPHEREPOSITION', 'DBL_ARRAY', [0.0, 0.0, 0.0])
     # Dielectric profile type
     # Valid for: SPHERICALDIFFUSE
-    # Valid values: TANH, ERF
+    # Valid values: TANH, ERF, LOG
     # Default: TANH
     green.add_kw('PROFILE', 'STR', 'TANH')
```

Comment:
Should the default be changed to `LOG` rather than `TANH`? If yes, change it here. Plus you will need to add a note in the changelog under the `Changed` section **and** update the documentation files:
1.  `doc/code-reference/greens-functions.rst`
2. `doc/users/input.rst`, in the `Profile` section.